.github/workflows/e2e-matrix.yaml

```diff
@@ -31,6 +34,11 @@ on:
           - "us-west-2"
           - "eu-west-1"
         default: "us-east-2"
+      local_zone_region:
+        type: choice
+        options:
+          - "us-west-2"
+        default: "us-west-2"
```

Comment:
If we're only trying to do us-west-2, do we need to parameterize this? Much like soak tests, we can just ensure that we're using the right input here rather than making a separate variable.

Reply:
My thought was for people running in this in PRs that may be running against different regions but it probably doesn't matter. Either way you'd have to manually extend the drop-down. Probably fine to hard-code this now and pull it out down the road if needed.

Reply:
+1, let's simplify this now unless we think there is a reason to add the choice here